src/KBS.Infrastructure/Configuration.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace KBS.Infrastructure
```

Comment:
This file should be moved to the FauxApplication project, it also should be an interface. The properties should be auto-properties.